easybuild/easyconfigs/u/UDUNITS/UDUNITS-2.2.25.eb

```diff
@@ -0,0 +1,27 @@
+# jgp@cscs
+easyblock = 'ConfigureMake'
+
+name = 'UDUNITS'
+version = '2.2.25'
+
+homepage = 'http://www.unidata.ucar.edu/software/udunits/'
+description = """UDUNITS supports conversion of unit specifications between formatted and binary forms,
+ arithmetic manipulation of units, and conversion of values between compatible scales of measurement."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+toolchainopts = {'opt': True, 'pic': True}
+
+source_urls = ['ftp://ftp.unidata.ucar.edu/pub/udunits']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('expat', '2.1.0')]
```

Comment:
why not using https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/e/expat/expat-2.2.0.eb ?

Reply:
Done.